### ISSUE TITLE
"\xE2" from ASCII-8BIT to UTF-8 (Encoding::UndefinedConversionError)

### ISSUE THREAD

**Issue by crevete** [aespinosa/minitest-junit#11]
Hello,

After upgrading to 2.0.0, when I run `rails test test/my_test.rb --junit`, I got this error when calling `Ox.dump(doc)`:

```
[...]/minitest-junit-2.0.0/lib/minitest/junit.rb:50:in `write': "\xE2" from ASCII-8BIT to UTF-8 (Encoding::UndefinedConversionError)
```

I found a workaround to fix this issue by forcing encoding of the `Ox::Document` object to UTF-8 during initialization:

```ruby
doc = Ox::Document.new(version: '1.0', encoding: 'UTF-8')
```

@aespinosa Can you please patch this ASAP? Thanks

**Comment by davidwessman** [comment]
That sound reasonable!

I am not able to do anything before Monday, so better downgrade and then write a PR with a test 😊

**Comment by aespinosa** [comment]
fixed in #12. thanks for the PR!
